playbooks/roles/shadowsocks/tasks/main.yml

```diff
@@ -91,6 +91,13 @@
   command: cat {{ shadowsocks_password_file }}
   register: shadowsocks_password
 
+- name: Copy the Shadowsocks password file locally
```

Comment:
That's a good idea. I don't think there's any benefit outside of Vagrant for the time being. I'll have to add a var that gets set for vagrant runs as well but that will likely end up being useful anyway~!